### ISSUE TITLE
tabs: improve accessibility

### ISSUE THREAD

**Issue by thbt** [ng-bootstrap/ng-bootstrap#1430]
### Bug description

Disabled tabs are not announced as disabled and should not be focusable.

### Solution

The easy fix would be to add `aria-dsabled="true"` and `tabindex="-1"` to the tab title link or replacing it by a `span` element.

However it is not clear yet how bootstrap will address this issue as they are thinking about removing ARIA attributes [twbs/bootstrap#19738](https://github.com/twbs/bootstrap/issues/19738)
